Description: disable extra fonts, we do not ship them at the moment
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: not-needed
Last-Update: 2016-02-14

--- a/unpacked/extensions/MathMenu.js
+++ b/unpacked/extensions/MathMenu.js
@@ -1573,13 +1573,7 @@
           ITEM.RADIO(["TeXImage","TeX (image)"],   "font", {action: MENU.Font}),
           ITEM.RULE(),
           ITEM.RADIO(["STIXLocal","STIX (local)"], "font", {action: MENU.Font}),
-          ITEM.RADIO(["STIXWeb","STIX (web)"], "font", {action: MENU.Font}),
-          ITEM.RULE(),
-          ITEM.RADIO(["AsanaMathWeb","Asana Math (web)"], "font", {action: MENU.Font}),
-          ITEM.RADIO(["GyrePagellaWeb","Gyre Pagella (web)"], "font", {action: MENU.Font}),
-          ITEM.RADIO(["GyreTermesWeb","Gyre Termes (web)"], "font", {action: MENU.Font}),
-          ITEM.RADIO(["LatinModernWeb","Latin Modern (web)"], "font", {action: MENU.Font}),
-          ITEM.RADIO(["NeoEulerWeb","Neo Euler (web)"], "font", {action: MENU.Font})
+          ITEM.RADIO(["STIXWeb","STIX (web)"], "font", {action: MENU.Font})
         ),
         ITEM.SUBMENU(["ContextMenu","Contextual Menu"],    {hidden:!CONFIG.showContext},
           ITEM.RADIO(["MathJax","MathJax"], "context"),
--- a/unpacked/jax/output/HTML-CSS/jax.js
+++ b/unpacked/jax/output/HTML-CSS/jax.js
@@ -432,11 +432,6 @@
       TeXImage:       ["",""],
       STIXLocal:      ["STIX","STIX-Web"],
       STIXWeb:        "STIX-Web",
-      AsanaMathWeb:   "Asana-Math",
-      GyrePagellaWeb: "Gyre-Pagella",
-      GyreTermesWeb:  "Gyre-Termes",
-      LatinModernWeb: "Latin-Modern",
-      NeoEulerWeb:    "Neo-Euler"
     },
     
     fontInUse: "generic",
--- a/unpacked/jax/output/SVG/jax.js
+++ b/unpacked/jax/output/SVG/jax.js
@@ -159,8 +159,7 @@
 
     hideProcessedMath: true,           // use display:none until all math is processed
 
-    fontNames: ["TeX","STIX","STIX-Web","Asana-Math",
-                "Gyre-Termes","Gyre-Pagella","Latin-Modern","Neo-Euler"],
+    fontNames: ["TeX","STIX","STIX-Web","Asana-Math"],
 
 
     Config: function () {
